ironfish/src/network/peers/peerConnectionManager.ts

```diff
@@ -84,24 +84,24 @@ export class PeerConnectionManager {
 
     let connectAttempts = 0
 
-    for (const peerCandidateIdentity of this.peerManager.peerCandidates.shufflePeerCandidates()) {
+    for (const [
+      peerCandidateIdentity,
+      peerCandidate,
+    ] of this.peerManager.peerCandidates.shufflePeerCandidates()) {
       if (connectAttempts >= CONNECT_ATTEMPTS_MAX) {
         break
       }
 
       if (!this.peerManager.identifiedPeers.has(peerCandidateIdentity)) {
-        const val = this.peerManager.peerCandidates.get(peerCandidateIdentity)
-        if (val) {
```

Comment:
this check is basically not required because the peercandidateidentity is in the peer candidate list (which we are iterating over)